templates/centos-7.2-ppc64-openstack.json

```diff
@@ -0,0 +1,57 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "c<wait5><wait10>",
+        "linux /ppc/ppc64/vmlinuz ro ",
+        "ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/centos-7.2/ks-ppc64-openstack.cfg<enter>",
+        "initrd /ppc/ppc64/initrd.img<enter>",
+        "boot<enter><wait>"
+      ],
+      "accelerator": "kvm",
+      "boot_wait": "6s",
+      "disk_size": 3072,
+      "headless": true,
+      "vnc_bind_address":"0.0.0.0",
```

Comment:
Unless this is specified to openpower1's v1.1.0-dev, packer seems to start vnc bound only 127.0.0.1.